src/lib/builders/popover/create.ts

```diff
@@ -213,14 +202,13 @@ export function createPopover(args?: CreatePopoverProps) {
 			} as const;
 		},
 		action: (node: HTMLElement): MeltActionReturn<PopoverEvents['trigger']> => {
+			activeTrigger.set(node);
```

Comment:
While this seems like a nice approach, it's limiting. Currently, we can use any arbitrary element as the `activeTrigger` using the `id`, this now turns using the builder into a requirement which is unideal for composition.

Reply:
I feel like letting people use the library without using the trigger builder is weird. The point of using melt is to have all the accessibility and functionality taken care of by the library. If you choose to not use the trigger builder, you are not going to have all the correct aria attributes, etc. so it seems awkward to be building the codebase around that use-case. of course, I may be totally wrong

Reply:
Letting people use any arbitrary element as the trigger without using our builder introduces so many edge cases. We always want to make sure we have the up-to-date reference to the trigger for the floating positioning. 

If we require using our trigger builder, we easily set the active trigger on trigger mount and set it to null on destroy. However, if we allow any arbitrary element to be used, how do we know if the element is still in the DOM? What if the trigger was unmounted and remounted. This will get very ugly very quickly if we want to actually fully support that and fix all the issues that will come with it.

Reply:
I agree and disagree with both of you 😄 

I agree with @anatolzak in the sense that making the id the implicit way of using it is weird. Two ways of doing things and all that.

However, Melt was indeed made to be super flexible. It provides accessible defaults, but allows you to step out of the reigns here and there. So if we want to support this, which I agree with @huntabyte that we should, we should just expose the `activeTrigger` store and document it, say it's dangerous, etc. This way its the users responsibility that things are breaking.